tasks/install.yml

```diff
@@ -53,12 +53,16 @@
 - name: REDIS | install required libs
```

Comment:
One thingy: all these "install something" tasks, start with a lowercase, when the other ones start with an Uppercase :grin: